source/mir/ndslice/selection.d

```diff
@@ -1534,6 +1534,7 @@ pure nothrow unittest
 
 /++
 Returns a slice, the elements of which are equal to the initial index value.
+This is multidimensional analog of $(LINK2 std_range.html#iota, std.range.iota).
```

Comment:
Maybe you could add a see also to iota slice? 
Moreover I think that if you at the multidimensional analog of iota, one might think about iotaSlice, so maybe pointing out that the index of reseted for every dimension could help to solve the confusion? 
And then "for a continuous index, see iotaSlice"